src/ca65/instr.c

```diff
@@ -783,7 +788,7 @@ static const InsTable* InsTabs[CPU_COUNT] = {
     (const InsTable*) &InsTab65816,
     (const InsTable*) &InsTabSweet16,
     (const InsTable*) &InsTabHuC6280,
-    0,                                  /* Mitsubishi 740 */
+    0,
```

Comment:
I think that the comment should stay there.  Otherwise, a reader might wonder, "Huh, why is that NULL there?"  Or, he might think that the list needs to be terminated by a NULL pointer.